plugin-server/tests/main/ingestion-queues/session-recording/session-recordings-consumer-v2.test.ts

```diff
@@ -7,23 +7,24 @@ import { defaultConfig } from '../../../../src/config/config'
 import { SessionRecordingIngesterV2 } from '../../../../src/main/ingestion-queues/session-recording/session-recordings-consumer-v2'
 import { Hub, PluginsServerConfig } from '../../../../src/types'
 import { createHub } from '../../../../src/utils/db/hub'
-import { createIncomingRecordingMessage } from './fixtures'
+import { getFirstTeam, resetTestDatabase } from '../../../helpers/sql'
+import { createIncomingRecordingMessage, createKafkaMessage, createTP } from './fixtures'
 
-const keyPrefix = 'test-session-offset-high-water-mark'
-
-async function deleteKeysWithPrefix(hub: Hub, keyPrefix: string) {
+async function deleteKeysWithPrefix(hub: Hub) {
     const redisClient = await hub.redisPool.acquire()
-    const keys = await redisClient.keys(`${keyPrefix}*`)
+    const keys = await redisClient.keys(`@posthog/replay/*`)
```

Comment:
this was "name-spaced" so that running the tests on a dev machine doesn't break local running... did it have to change?

Reply:
Hmmm I can probably revert. It was just getting annoying to manage across all the different implementers...